docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

```diff
@@ -85,6 +85,10 @@ Args syntax
      - true
      - String
      - Full url of webhook
+   * - webhook_from_env
+     - true
```

Comment:
Could you change the `required` column to `false` for both `webhook` and `webhook_from_env` ?

Reply:
done.